src/plugins/mailer/builder.ts

```diff
@@ -0,0 +1,169 @@
+import { TFunction } from 'i18next';
+
+import { DEFAULT_LANG } from '@graasp/translations';
+
+import { GRAASP_LANDING_PAGE_ORIGIN } from '../../utils/constants';
+import i18next from './i18n';
+import { MAIL } from './langs/constants';
+import { Mail } from './service';
+
+type CssStyles = { [key: string]: string };
+
+export class MailBuilder {
+  private subject: string;
+  private plainText = '';
+  private htmlText = '';
```

Comment:
Theses 2 properties are really confusing to me. I've understood that they're are strongly related, so is it possible to have something that represent that, or having a comment to give some explanation ?

It would be awesome (maybe a bit difficult), to have a private function `appendText(...)` who is the only responsible of the content of theses properties. It could eventually reduce the amount of this file's lines

Reply:
I have added comments that explain the need for these two separate properties.